test/server/chat-plugins/youtube.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Tests for the Youtube room plugin.
+ * Written by mia-pi.
+ */
+'use strict';
+const YoutubeInterface = require('../../../.server-dist/chat-plugins/youtube').YoutubeInterface;
+const assert = require('../../assert').strict;
```

Comment:
```suggestion
const assert = require('../../assert');
```

I'm surprised `.strict` exists at all, but it clearly shouldn't be used here. Our assert library is already strict.